lib/dynflow/rails/daemon.rb

```diff
@@ -42,23 +64,19 @@ def run_background(command = 'start', options = {})
         STDOUT.puts("Dynflow Executor: #{command} in progress")
 
         options[:executors_count].times do
-          Daemons.run_proc(options[:process_name],
-                           :multiple => true,
-                           :dir => options[:pid_dir],
-                           :log_dir => options[:log_dir],
-                           :dir_mode => :normal,
-                           :monitor => true,
-                           :log_output => true,
-                           :ARGV => [command]) do |*_args|
-                             begin
-                               ::Logging.reopen
-                               run(options[:rails_root])
-                             rescue => e
-                               STDERR.puts e.message
-                               ::Rails.logger.exception('Failed running Dynflow daemon', e)
-                               exit 1
-                             end
-                           end
+          daemons_class.run_proc(
+            options[:process_name],
+            daemons_options(command, options)
+          ) do |*_args|
+            begin
+              ::Logging.reopen
+              run(options[:rails_root], options)
```

Comment:
Oh, `dynflow-executor` and `foreman-tasks` each pass different options, this needs to be unified either here or in the scripts